charms/istio-pilot/tox.ini

```diff
@@ -21,4 +21,4 @@ paths = {toxinidir}/src {toxinidir}/tests
 [testenv:lint]
 commands =
     flake8 {[vars]paths}
-    black --check {[vars]paths}
+    black --check --diff {[vars]paths}
```

Comment:
This PR is for adding SSL support, please push another PR for adding changes to `tox.ini` and related.